src/pages/academy/grading/subcomponents/GradingEditor.tsx

```diff
@@ -46,8 +46,8 @@ type OwnProps = {
   initialXp: number;
   xpAdjustment: number;
   maxXp: number;
-  studentName: string;
-  studentUsername: string;
+  studentName: string | undefined;
+  studentUsername: string | undefined;
```

Comment:
Seems like these will always be defined? Is there a case I'm missing where they will not be defined?

Reply:
Yup they will always be defined. Previously undefined to suppress the errors from team submission changes!

Reply:
Resolved.